media/inf/syssetup.inf.tpl

```diff
@@ -1,15 +1,12 @@
-;
 ; For Windows, this file controls various aspects of the Installation and
 ; Upgrade process. If you know a good documentation about this file,
 ; please add a link here.
-;
-;
+
```

Comment:
Please revert one ';' on a newline.

Reply:
With which intention? What is it used for?